db/structure.sql

```diff
@@ -120,4 +120,74 @@ CREATE TABLE `settings` (
   PRIMARY KEY (`id`),
   INDEX `idx_key` (`key`)
 ) ENGINE=InnoDB AUTO_INCREMENT=101 DEFAULT CHARSET=utf8mb4;
-/*!40101 SET character_set_client = @saved_cs_client */;
\ No newline at end of file
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+
+--
+-- Table structure for table `capabilities`
+--
+
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `capabilities` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `description` varchar(255) NOT NULL,
+  `is_system_supplied` int(11) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb4;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+
+--
+-- Table structure for table `groups`
+--
+
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `groups` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `description` varchar(255) NOT NULL,
+  `is_system_supplied` int(11) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8mb4;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+
+--
+-- Table structure for table `people_groups`
+--
+
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `people_groups` (
+  `people_id` varchar(255) NOT NULL,
+  `group_id` varchar(255) NOT NULL,
+  KEY `idx_people_id` (`people_id`),
+  KEY `idx_group_id` (`group_id`)
```

Comment:
The primary key in this case is the relationship between the `group_id` and the `people_id`.
```suggestion
 PRIMARY KEY(`people_id`, `group_id`)
  KEY `idx_people_id` (`people_id`),
  KEY `idx_group_id` (`group_id`)
```